### ISSUE TITLE
New export not reflected in webref

### ISSUE THREAD

**Issue by jyasskin** [w3c/webref#1071]
https://github.com/w3c/webdriver-bidi/pull/591/files added `data-export` to the ["extension modules" definition](https://w3c.github.io/webdriver-bidi/#extension-modules), and it was [published](https://github.com/w3c/webdriver-bidi/actions/runs/6771825202) to https://w3c.github.io/webdriver-bidi/#extension-modules about 2023-11-06T14:03:11Z. Webref was [most recently updated](https://github.com/w3c/webref/commit/72d4a5479ea511863bcff4fa08daf22e3e3a604e) at 2023-11-07T17:05:15Z, but it didn't update https://github.com/w3c/webref/blob/main/ed/dfns/webdriver-bidi.json, which still says the definition is `"access": "private"`. Any idea what went wrong?

**Comment by thiagowfx** [comment]
I also just landed https://github.com/w3c/webdriver-bidi/pull/593 that exports more definitions. We could maybe use it as another data point to check whether it's an specific issue with https://github.com/w3c/webdriver-bidi/pull/591 or with webref. Let's wait until tomorrow.

**Comment by tidoust** [comment]
Yes, sorry, that was a side effect of a bug in the version of the crawler that I released yesterday. In practice, the crawl [errored on all specs](https://github.com/w3c/webref/blob/f3042b9c969c1d296a95e98b4c6570412f5d606c/ed/index.json#L614) and thus failed to update any data.

Bug was fixed in https://github.com/w3c/reffy/pull/1414. I just ran a new crawl. [Updates of WebDriver Bidi's definitions](https://github.com/w3c/webref/commit/9e34520de1c52506331e8862c0e9a4588da0356b#diff-f9905ed052b46ab53de92919ec7aca25605c4dc9c05a1b6b73c3072a616f4569) seem to have been correctly picked up.

Closing this as a result. Let me know if I missed something!
